src/solver/conv_wino_fury_RxS.cpp

```diff
@@ -171,6 +171,10 @@ bool ConvWinoFuryRxS<Winodata, Winofilter>::IsApplicable(const ExecutionContext&
     if(!problem.Is2d())
         return false;
 
+    if(problem.HasNonPackedTensors())
+    {
+        return false;
+    }
```

Comment:
This solver works fine with strides too.

Reply:
adding back the check to WinoGrad solvers at @atamazov 's recommendation